### PR TITLE
Added FJLabs Polaris PCB's

### PR DESCRIPTION
**Pull request by mrnoisytiger**
## Description

Added FJLabs Polaris PCB's 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mrnoisytiger** [comment]
New pull request to fix submodules problem of https://github.com/qmk/qmk_firmware/pull/12325

**Review by fauxpark — APPROVED** [review]

File: keyboards/fjlabs/polaris/rules.mk [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+KEYBOARD_LOCK_ENABLE = yes
```
Comment (drashna) [inline keyboards/fjlabs/polaris/rules.mk:26]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by mrnoisytiger** [comment]
@drashna Changed!

File: keyboards/fjlabs/polaris/readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Polaris-Compatible Hotswap and Solder PCB's
+
+The following is the QMK Firmware for the Polaris-Compatible Hotswap and Solder PCB's developed by FJLaboratories (FJLabs).
+* Keyboard Maintainer: FJLabs
+* Hardware Supported: Polaris Hotswap and Solder
+ 
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/fjlabs/polaris/readme.md:7]: Could you add instructions on how to jump to bootloader?

Comment (mrnoisytiger) [reply keyboards/fjlabs/polaris/readme.md:7]: Added

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**